internal/cloud/backup.go

```diff
@@ -16,4 +16,10 @@ type Backup struct {
 	// VaultName is the identifier of the place in the cloud where the archive was
 	// stored.
 	VaultName string
+
+	// Paths are the folders that were synchronized with this backup.
+	Paths []string
```

Comment:
The reason to keep the paths tracked in the backup is to easily detect differences between the local files and what was uploaded to the cloud (#12), specially when the backup path changes. But this could also be achieved with a control file.